.github/workflows/release.yml

```diff
@@ -267,9 +277,15 @@ jobs:
           name: ArduinoCreateAgent-osx
           path: ArduinoCreateAgent-osx
 
+        # workaround to strip bugfix number from semver (only to make 1.1 release) I will change this in the future
+      - name: Set version env vars
+        # VERSION will be available only in the next step
+        run: |
+          echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+
```

Comment:
Unfortunately the trick of using `$GITHUB_ENV` do not work between different jobs